test/integration/match2.test.ts

```diff
@@ -170,53 +227,6 @@ describe('match2', () => {
       })
     })
 
-    it('should acceptA then acceptFinal a match2 on-chain', async () => {
-      match2AcceptMock()
-      // submit to chain
-      const responseAcceptA = await post(app, `/match2/${seededMatch2Id}/accept`, {})
-      expect(responseAcceptA.status).to.equal(201)
-
-      // check local entities update with token id
-      const [match2AcceptA] = await db.getMatch2(seededMatch2Id)
-      expect(match2AcceptA.latestTokenId).to.equal(match2AcceptMockTokenId)
-      expect(match2AcceptA.state).to.equal(Match2State.acceptedA)
-      expect(match2AcceptA.originalTokenId).to.equal(seededMatch2TokenId)
-
-      match2AcceptFinalMock()
-      // submit to chain
-      const responseAcceptFinal = await post(app, `/match2/${seededMatch2Id}/accept`, {})
-      expect(responseAcceptFinal.status).to.equal(201)
-
-      // check local entities update with token id
-      const [demandA] = await db.getDemand(seededOrderId)
-      expect(demandA.latestTokenId).to.equal(match2AcceptFinalMockTokenIds[0])
-      expect(demandA.state).to.equal(DemandState.allocated)
-      expect(demandA.originalTokenId).to.equal(seededOrderTokenId)
-
-      const [demandB] = await db.getDemand(seededCapacityId)
-      expect(demandB.latestTokenId).to.equal(match2AcceptFinalMockTokenIds[1])
-      expect(demandB.state).to.equal(DemandState.allocated)
-      expect(demandB.originalTokenId).to.equal(seededCapacityTokenId)
-
-      const [matchAcceptFinal] = await db.getMatch2(seededMatch2Id)
-      expect(matchAcceptFinal.latestTokenId).to.equal(match2AcceptFinalMockTokenIds[2])
-      expect(matchAcceptFinal.state).to.equal(Match2State.acceptedFinal)
-      expect(matchAcceptFinal.originalTokenId).to.equal(seededMatch2TokenId)
-    })
-
-    it('should acceptB a match2 on-chain', async () => {
-      match2AcceptMock()
-      // submit to chain
-      const response = await post(app, `/match2/${seededMatch2OrderNotOwnedId}/accept`, {})
-      expect(response.status).to.equal(201)
-
-      // check local entities update with token id
-      const [match2] = await db.getMatch2(seededMatch2OrderNotOwnedId)
-      expect(match2.latestTokenId).to.equal(match2AcceptMockTokenId)
-      expect(match2.state).to.equal(Match2State.acceptedB)
-      expect(match2.originalTokenId).to.equal(seededMatch2TokenId)
-    })
```

Comment:
this is the test scenario that will require mocking `ChainNode` because the `demand-create` process flow makes it impossible to put a demand on chain that's not owned by the current user